assets/src/dashboard/app/api/useStoryApi.js

```diff
@@ -250,7 +273,15 @@ const useStoryApi = (dataAdapter, { storyApi, encodeMarkup }) => {
           },
           flags,
         });
-        const response = await dataAdapter.post(storyApi, {
+
+        const path = queryString.stringifyUrl({
+          url: storyApi,
+          query: {
+            _fields: ['edit_link'].join(','),
```

Comment:
Why do this?

Reply:
No reason other than that it's easy to extend and follows the same pattern elsewhere.